torchtext/csrc/vocab.cpp

```diff
@@ -20,24 +20,24 @@ Vocab::Vocab(const StringList &tokens, const std::string &unk_token)
       throw std::runtime_error("Duplicate token found in tokens list: " +
                                tokens[i]);
     }
-    stoi_.insert(std::move(tokens[i]), i);
+    stoi_[std::move(tokens[i])] = i;
```

Comment:
I'm wondering if we should limit the scope of the PR and just change the variable name. To debug the windows failure, we might move `insert`, `->value()` etc back. We just need to limit the scope of the PR if we want to debug the wire discrepancy across different platforms.